ejb3/src/main/java/org/jboss/as/ejb3/logging/EjbLogger.java

```diff
@@ -408,6 +408,10 @@
     @Message(id = 37, value = "Dynamic stub creation failed for class %s")
     void dynamicStubCreationFailed(String clazz, @Cause Throwable t);
 
+    @LogMessage(level = INFO)
+    @Message(id = 38, value = "Jakarta Enterprise Beans business method %s must be public")
+    void ejbBusinessMethodMustBePublic(final Method method);
+
```

Comment:
Not sure if it's good to reuse the log id `38`. But if we decide to log it in debug level, we can just directly log it in java class without this entry, so this won't be a problem.

Reply:
Fixed. Moved at the end with the new number.